automated install/basic-install.sh

```diff
@@ -1281,7 +1282,10 @@ version_check_dnsmasq() {
         #
         sed -i '/^server=@DNS2@/d' ${dnsmasq_pihole_01_location}
     fi
-
+	
+	# Set the cache size
+	sed -i "s/@CACHE_SIZE@/$CACHE_SIZE/" ${dnsmasq_pihole_01_location}
```

Comment:
If the following is used, 01-pihole.conf will not need to be modified.
`sed -i 's/\(cache-size=\).*/\1${CACHE_SIZE}/' ${dnsmasq_pihole_01_location}`

Reply:
The PR keeps in the style of the rest of the code as it exists.

Reply:
I indeed followed the rest of the code style, I'm not really that knowledgeable about shell scripts   but can make changes (as long as I understand how :)) if anything is needed.

Reply:
It looks good to me.